client/src/providers/AuthProvider.tsx

```diff
@@ -22,7 +22,7 @@ export interface AuthContext {
   loadMeQuery: (variables: AuthProviderMeQueryVariables, options?: any) => void;
 }
 
-const [useCtx, Provider] = createCtx<AuthContext>();
+export const [useCtx, AuthContextProvider] = createCtx<AuthContext>();
```

Comment:
I see you exported `AuthContextProvider` to use it only inside tests. An alternative way of overriding `AuthContext` is to mock `useAuthContext` hook.